docs/mlir_interoperation.ipynb

```diff
@@ -166,11 +165,9 @@
      "output_type": "stream",
      "text": [
       "\"builtin.module\"() ({\r\n",
-      "  %0 = \"arith.constant\"() {value = 1 : i32} : () -> i32\r\n",
-      "  %1 = \"arith.constant\"() {value = 2 : i32} : () -> i32\r\n",
+      "  %0 = \"arith.constant\"() <{value = 1 : i32}> : () -> i32\r\n",
+      "  %1 = \"arith.constant\"() <{value = 2 : i32}> : () -> i32\r\n",
       "  %2 = \"arith.addi\"(%0, %1) : (i32, i32) -> i32\r\n",
-      "  %3 = \"arith.addi\"(%2, %2) : (i32, i32) -> i32\r\n",
```

Comment:
Is there a reason why you removed this too? I am fine with it, but this triggered the pattern matching in my brain :sweat_smile:

Reply:
I can't find one :thinking: Good catch :yum: